src/v2/styles/Message/Message-theme.scss

```diff
@@ -226,6 +229,9 @@
       @include utils.button-reset;
       border-radius: var(--str-chat__message-options-border-radius);
 
+      width: var(--str-chat__message-options-button-size);
+      height: var(--str-chat__message-options-button-size);
```

Comment:
Shouldn't this be inside the layout file?

Reply:
I think that this particular piece is theme rather than layout (as in, font size). But if it is inconsistent, I can take it there.

Reply:
To me, it seems more consistent with the current setup inside the layout file because all other icon sizes are set inside the layout files